executor/callback/callback_client.py

```
@@ -170,6 +170,18 @@ def _do_send_callback(self, data: Dict[str, Any]) -> Dict[str, Any]:
         # Prepare headers with trace context for distributed tracing
         headers = {"Content-Type": "application/json"}
+
+        # Inject X-Request-ID for log correlation
+        try:
+            from shared.telemetry.context import get_request_id
+
+            request_id = get_request_id()
+            if request_id:
+                headers["X-Request-ID"] = request_id
+        except Exception:
+            pass
```

Ruff (0.14.13)

`try`-`except`-`pass` detected, consider logging the exception

(S110)